### PR TITLE
*: bump to v0.11.0+git for v0.11.x

### PR DESCRIPTION
**Pull request by hasbro17**
**Description of the change:**
Post-release branch bump for v0.10.x

**Motivation for the change:**
Versioning

### CODE REVIEW

File: internal/pkg/scaffold/gopkgtoml.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ const gopkgTomlTmpl = `[[override]]
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  # branch = "master" #osdk_branch_annotation
-  version = "=v0.11.0" #osdk_version_annotation
+  branch = "v0.11.0" #osdk_branch_annotation
```
Comment (joelanford) [inline internal/pkg/scaffold/gopkgtoml.go]: v0.11.x?

**Comment by hasbro17** [comment]
I thought `require github.com/operator-framework/operator-sdk v0.11.x` is how you'd point to the branch like we did before. Not sure if that's invalid or if that branch isn't in the proxy cache.
```console
invalid module version "v0.11.x": unexpected status (https://proxy.golang.org/github.com/operator-framework/operator-sdk/@v/v0.11.x.info): 410 Gone
```

**Comment by joelanford** [comment]
@hasbro17 Hmm. That's a strange one. I did a couple of things to troubleshoot.

1. I tried using go 1.12 with GOPROXY and `v0.11.x` (what this PR should be doing) and it DOES NOT resolve.
2. I tried using go 1.12 without GOPROXY and `v0.11.x` and it correctly resolves.
3. I tried using go 1.12 with GOPROXY and `v0.10.x` and it correctly resolves.
4. I tried using go 1.13 with GOPROXY and `v0.11.x` and it correctly resolves.


It seems like whatever the problem is, it is fixed in go 1.13. There could be a couple of things at play here:
* `v0.11.x` and `v0.11.0` currently point to the same commit. Maybe that's part of the problem?
* `v0.11.x` looks a lot like a semver string, so perhaps Go is getting confused on it (that doesn't necessarily explain why `v0.10.x` works with go1.12, unless there are multiple conditions that together cause the bug)

Maybe try some things out in your fork?

**Comment by joelanford** [comment]
/hold 

openshift-bot is determined to get this working 🤣

**Pull request merged**